src/adminEntity.ts

```diff
@@ -6,6 +6,7 @@ import ManyToManyWidget from './widgets/manyToManyWidget'
 
 abstract class AdminEntity {
   abstract entity: EntityType
+  listDisplay: string[] | null = null
```

Comment:
This should provide better typing

```suggestion
  listDisplay: Array<keyof EntityType> | null = null
```